site-to-site-vpn/README.md

```diff
@@ -0,0 +1,39 @@
+# Site to Site VPN Connection
+
+<a href="https://azuredeploy.net/" target="_blank">
```

Comment:
The deploy button code needs to have the Ibiza portal deeplink. See here 

https://raw.githubusercontent.com/Azure/azure-quickstart-templates/2d4665806c105354b2ea9dccc19ad5b8b952579e/101-create-key-vault/README.md